src/config/dark-sites.config

```diff
@@ -602,6 +603,7 @@ migueldemoura.com
 mikemaximus.github.io/gbm-web
 minehut.com
 miniroyale2.io
+minlor.net/$
```

Comment:
Why the `$`?

Reply:
I got a little bit confused sorry about that. removed it now